apps/site-nft/src/components/App/index.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ *
+ * App.js
+ *
+ * This component is the skeleton around the actual pages, and should only
+ * contain code that should be seen on all pages. (e.g. navigation bar)
+ *
+ */
+
+import * as React from 'react';
+import { MainNavigation } from '../MainNavigation';
+import { Footer } from 'components/Footer';
+import { MainPageTransition } from '@thecointech/site-base/components/MainPageTransition';
+import { MainRouter } from '../MainRouter';
+import { useAccountStoreApi } from '@thecointech/shared/containers/AccountMap';
+import { addDevLiveAccounts } from '../../api/mock/accounts';
+
+// Either import CSS or LESS;
+// - LESS is slower, but offers on-save hot-reload
+// - CSS is faster, but requires manual recompile
+import '../../semantic/semantic.css';
+//import '@thecointech/site-semantic-theme/semantic.less';
+import styles from './styles.module.less';
+
+
+export const App = () => {
+
+  if (process.env.NODE_ENV === 'development') {
+    //useDevPrep(account)
+    if (process.env.CONFIG_NAME === 'devlive') {
+      const accountsApi = useAccountStoreApi();
+      React.useEffect(() => { addDevLiveAccounts(accountsApi) })
+    }
+  }
+
```

Comment:
This is where things start deviating from the original.  This app is structured a little bit differently from the original.  Here we trial using the css `grid` layout instead of the div's in the original.  The content is structured in 3 parts: top/middle/botton.